tests/composite_pk/test_filter.py

```diff
@@ -78,6 +79,16 @@ def test_order_comments_by_pk_desc(self):
             ),
         )
 
+    def test_order_comments_by_pk_expr(self):
+        self.assertQuerySetEqual(
+            Comment.objects.order_by("pk"),
+            Comment.objects.order_by(F("pk")),
+        )
+        self.assertQuerySetEqual(
+            Comment.objects.order_by("-pk"),
+            Comment.objects.order_by(F("pk").desc(nulls_last=True)),
```

Comment:
`nulls_last` is a noop since primary key cannot contain nulls but I figured I'd add it to test the emulation layer for backends that lack support for it.